ReactAndroid/src/main/java/com/facebook/react/views/toolbar/ReactToolbar.java

```diff
@@ -138,17 +156,20 @@ public void onFinishTemporaryDetach() {
     super.onFinishTemporaryDetach();
     mLogoHolder.onAttach();
     mNavIconHolder.onAttach();
+    mOverflowIconHolder.onAttach();
```

Comment:
Can you replace these three lines with a call to `attachDraweeHolders()`? Not sure how it ended up like this.